src/flamegraph/mod.rs

```diff
@@ -182,23 +185,27 @@ where
             a_extra = Some(&attrs.a.extra);
         }
 
-        svg.write_event(Event::Start({
-            let mut g = BytesStart::borrowed_name(b"g").with_attributes(args!(
-                "class" => class
-            ));
-            if let Some(style) = style {
-                g.extend_attributes(std::iter::once(("style", style)));
-            }
+        if let Event::Start(ref mut g) = event_start {
+            // clear the BytesStart
+            g.clear_attributes();
```

Comment:
You'll have to bump `quick-xml` to `0.13.3` in `Cargo.toml` for this to compile :)

Reply:
No need, a new cargo build downloads quick-xml 0.13.3 as in Cargo.toml we have `quick-xml = 0.13`

Reply:
I should probably update the Cargo.lock file though.

Reply:
I think you should explicitly set `0.13.3`, since we _require_ that version now.